packages/wallets/src/components/WalletsAddMoreCardBanner/WalletsAddMoreCardBanner.tsx

```diff
@@ -21,7 +21,7 @@ const WalletsAddMoreCardBanner: React.FC<TProps> = ({
     is_crypto: isCrypto,
     landing_company_name: landingCompanyName,
 }: TProps) => {
-    const { switchAccount } = useDerivAccountsList();
+    const { switchAccount } = useAuthorize();
```

Comment:
Better import `switchAccount` directly from `useAuthorize` (it's not being overloaded in `useDerivAccountsList`)